### PR TITLE
[RubyGems.org] Remove wildcard

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/GemCutter.org.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+		gemcutter.org
+
+-->
+<ruleset name="GemCutter.org">
+
+	<target host="gemcutter.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/GemCutter.org.xml]: Same for `www`.
The file should be referenced in `RubyGems.xml`.

Comment (Bisaloo) [reply src/chrome/content/rules/GemCutter.org.xml]: When you go through the invalid certificate warning for `www`:

> Fastly error: unknown domain: www.gemcutter.org. Please check that this domain has been added to a service.

File: src/chrome/content/rules/GemCutter.org.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+		gemcutter.org
```
Comment (J0WI) [inline src/chrome/content/rules/GemCutter.org.xml]: Could you also add a reference back to `RubyGems.xml`?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**